aten/src/ATen/native/native_functions.yaml

```diff
@@ -10170,23 +10170,23 @@
 # See linalg_det as an example.
 
 # "_ex" stands for experimental
-- func: linalg_cholesky_ex(Tensor self, *, bool check_errors=False) -> (Tensor L, Tensor info)
+- func: linalg_cholesky_ex(Tensor self, *, bool upper=False, bool check_errors=False) -> (Tensor L, Tensor info)
```

Comment:
This is actually an FC-breaking change but it's probably OK because this function is so new